### PR TITLE
Sunder multiplier

### PR DESCRIPTION
**Pull request by Barnet2**
## About The Pull Request
Sunder multiplier: multiplies incoming sunder by this amount.
Gives queen a sunder multiplier of 0.8, crusher 0.5.
## Why It's Good For The Game
Of the three armored castes (queen, crusher, king), crusher has the least of being able to fight without sunder. The other two castes have an ability to operate at range even with sunder, but Crusher is always in there. Queen doesn't exactly require it per-say, but since they are still somewhat of a sunder magnet they have it. King literally needs sunder to die, so no multiplier on his end.
## Changelog
:cl:
balance: Queen has a sunder multiplier of 0.8 (takes 80% sunder), Crusher has a sunder multiplier of 0.5 (takes 50% sunder)
/:cl:

### CODE REVIEW

**Comment by Naaanii** [comment]
Double buff is kinda iffy and pools exist. I think sunder cap is kind of neat but also the stronger buff.

**Comment by Arkansas-Bar-Association** [comment]
Max sunder of 50% is crazy, I understand regenerating fast but that's just gonna make crushers very oppressive

**Comment by Arturlang** [comment]
50% max already means you half of your ehp, crusher is very reliant on having armor, alternative could be hard armor

**Comment by Barnet2** [comment]
> 50% max already means you half of your ehp, crusher is very reliant on having armor, alternative could be hard armor

There should still be a way to make the Crusher weaker, but not useless.

**Comment by AusarAtkin** [comment]
Isnt 50 sunder max a bit too much?, and why not just give it defender's ability instead to heal sunder in a pinch?

**Comment by Looking4bros** [comment]
Hmmmm, Perhaps a reduction in Melee armor to 75 would be a fitting counterbalance...

**Comment by MiniMeatwad** [comment]
I've found 60% sunder was just right to make just about any non-AP weapon capable of killing a crusher.

**Comment by novaepee** [comment]
xeno golden age

**Comment by Woker2004** [comment]
the buff crusher needs is clearly diagonal charge due to caves :clueless:

**Comment by Kuro020** [comment]
you should edit the tittle and stuff to reflect the changes

File: code/modules/mob/living/carbon/xenomorph/castes/crusher/castedatum_crusher.dm [hunk 1]
```diff
@@ -35,6 +35,9 @@
 	// *** Defense *** //
 	soft_armor = list(MELEE = 90, BULLET = 75, LASER = 75, ENERGY = 75, BOMB = 130, BIO = 100, FIRE = 40, ACID = 100)
 
+	// *** Sunder *** //
+	sunder_multiplier = 0.5 //takes half sunder
```
Comment (TiviPlus) [inline code/modules/mob/living/carbon/xenomorph/castes/crusher/castedatum_crusher.dm:39]:
```suggestion
	sunder_multiplier = 0.5
```

File: code/modules/mob/living/carbon/xenomorph/castes/queen/castedatum_queen.dm [hunk 2]
```diff
@@ -23,6 +23,9 @@
 	// *** Health *** //
 	max_health = 500
 
+	// *** Sunder *** //
+	sunder_multiplier = 0.8 //takes 80% sunder
```
Comment (TiviPlus) [inline code/modules/mob/living/carbon/xenomorph/castes/queen/castedatum_queen.dm:27]:
```suggestion
	// *** Sunder *** //
	sunder_multiplier = 0.8
```

**Review by TiviPlus — CHANGES_REQUESTED** [review]

**Pull request merged**